internal/testutils/marshal_any.go

```diff
@@ -19,18 +19,30 @@ package testutils
 
 import (
 	"fmt"
+	"google.golang.org/protobuf/protoadapt"
+	"testing"
 
 	"github.com/golang/protobuf/proto"
-	"github.com/golang/protobuf/ptypes"
 	"google.golang.org/protobuf/types/known/anypb"
 )
 
 // MarshalAny is a convenience function to marshal protobuf messages into any
 // protos. It will panic if the marshaling fails.
-func MarshalAny(m proto.Message) *anypb.Any {
-	a, err := ptypes.MarshalAny(m)
+func MarshalAny(m proto.Message) (*anypb.Any, error) {
```

Comment:
Why does this have to return an error? This should be changed to accept a `testing.T` and directly call `t.Fatal`.

Reply:
Done